libs/app.bundle.js

```diff
@@ -17335,8 +17391,8 @@ module.exports = function(XMPP, eventEmitter)
                     {
                         var audioMuted = startMuted.attr("audio");
                         var videoMuted = startMuted.attr("video");
-                        APP.UI.setInitialMuteFromFocus((audioMuted === "true"),
-                            (videoMuted === "true"));
+                        eventEmitter.emit(XMPPEvents.START_MUTED_FROM_FOCUS,
```

Comment:
You can use "audioMuted" and "videoMuted" variables here or remove them. What do you think?